android/anc/src/test/java/org/smartregister/fhircore/anc/ui/reports/ReportHomeScreenTest.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2021 Ona Systems, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.smartregister.fhircore.anc.ui.reports
+
+import android.app.Application
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.test.core.app.ApplicationProvider
+import io.mockk.mockk
+import io.mockk.spyk
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.robolectric.annotation.Config
+import org.smartregister.fhircore.anc.data.report.ReportRepository
+import org.smartregister.fhircore.anc.robolectric.RobolectricTest
+import org.smartregister.fhircore.anc.shadow.AncApplicationShadow
+
+@Config(shadows = [AncApplicationShadow::class])
+class ReportHomeScreenTest : RobolectricTest() {
+
+  private val app = ApplicationProvider.getApplicationContext<Application>()
+  private lateinit var repository: ReportRepository
+  private lateinit var viewModel: ReportViewModel
+  @get:Rule val composeRule = createComposeRule()
+
+  @Before
+  fun setUp() {
+    repository = mockk()
+    // val dataItems = spyk<Encounter>()
+    // coEvery { viewModel.getReportsTypeList() } returns emptyList<>()
+    viewModel =
+      spyk(objToCopy = ReportViewModel(ApplicationProvider.getApplicationContext(), repository))
+    // composeRule.setContent { ReportHomeScreen(viewModel = viewModel) }
+  }
+
+  @Test
+  fun testReportHomeScreenComponents() {
```

Comment:
This test can be ignored with appropriate reason instead of commenting the code

Reply:
updated with ignore reason